src/components/contextual/pages/vebal/LMVoting/GaugeVoteModal.vue

```diff
@@ -81,23 +82,22 @@ const voteInputDisabled = computed((): boolean => {
   return !!voteError.value;
 });
 
-const currentWeight = computed(() => props.gauge.userVotes);
+const currentWeight = computed(() => props.pool.userVotes);
 const currentWeightNormalized = computed(() =>
   scale(bnum(currentWeight.value), -2).toString()
 );
 const hasVotes = computed((): boolean => bnum(currentWeight.value).gt(0));
 
-const isVeBalGauge = computed((): boolean =>
-  isSameAddress(props.gauge.address, VEBAL_VOTING_GAUGE?.address || '')
-);
-
 // Is votes next period value above voting cap?
 const votesNextPeriodOverCap = computed((): boolean => {
-  if (!isVeBalGauge.value && props.gauge.relativeWeightCap === 'null')
-    return false;
-  const gaugeVoteWeightNormalized = scale(props.gauge.votesNextPeriod, -18);
+  const gaugeVoteWeightNormalized = scale(props.pool.votesNextPeriod, -18);
+  if (isVeBalGauge.value) {
+    return gaugeVoteWeightNormalized.gte(bnum('0.1'));
```

Comment:
Why is this hard coded here instead of dynamically pulled like the rest? Can it be put into a const variable rather than a magic number.